Presentation.Web/Models/External/V2/ItInterfaceResponseDTO.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+namespace Presentation.Web.Models.External.V2
+{
+    public class ItInterfaceResponseDTO
+    {
+        /// <summary>
+        /// UUID for IT-Interface
+        /// </summary>
+        public Guid Uuid { get; set; }
+
+        /// <summary>
+        /// UUID for IT-System exposing this IT-Interface
+        /// </summary>
+        public Guid ExposedBySystemUuid { get; set; }
+        
+        /// <summary>
+        /// Name of IT-Interface
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Identifier for IT-Interface
+        /// </summary>
+        public string InterfaceId { get; set; }
+
+        /// <summary>
+        /// Version signifier for IT-Interface
+        /// </summary>
+        public string Version { get; set; }
+
+        /// <summary>
+        /// General description of the IT-Interface
+        /// </summary>
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Url reference for further information
+        /// </summary>
+        public string UrlReference { get; set; }
+
+        // TODO: Did we decide to include these?
+        //public bool Deactivated { get; set; }
```

Comment:
de er ikke i spec men det ville være underligt ikke at aflevere dem

Reply:
Så tager vi dem bare med

Reply:
Se anden note omkring samme ift IT-systemer